### PR TITLE
require units on cbg, smbg, wizard events

### PR DESCRIPTION
**Pull request by jebeck**
Here's the small builder change @kentquirk 

I made another small change because I happened to notice it - it was very bad that we were producing default `recommended` objects with values of 0 (because a 0 suggestion is a suggestion)! This change required a jellyfish change though: https://github.com/tidepool-org/jellyfish/pull/46

### CODE REVIEW

File: lib/objectBuilder.js [hunk 1]
```diff
@@ -110,11 +110,23 @@ module.exports = function () {
...
+    var rec = _.assign(_createObject(), deviceInfo, {
+      type: 'cbg',
+      value: REQUIRED,
+      units: REQUIRED
+    });
+    rec._bindProps();
+    return rec;
   }
 
   function makeNote() {
```
Comment (kentquirk) [inline lib/objectBuilder.js]: Now that _makeWithValue is used in only one place, could you also just remove it from makeNote and eliminate the function?

**Comment by kentquirk** [comment]
Tests are now failing. Please fix?

File: lib/objectBuilder.js (hunk 1 above)
Comment (jebeck) [inline lib/objectBuilder.js]: Yeah, can do.

**Comment by kentquirk** [comment]
I piled on the changes to use the new units behavior in dexcom and abbott.

**Pull request merged**